rebasehelper/specfile.py

```diff
@@ -113,7 +113,7 @@ def wrapper(spec, *args, **kwargs):
 
 class SpecFile:
 
-    """Class representing a SPEC file"""
+    """Class representing a SPEC file. **Using SpecFile wipes all macros.**"""
```

Comment:
I'm afraid this is not formulated too well. How about: `Be aware that SpecFile modifies RPM macros in global context.`?

Reply:
Okay, I will change it :) I've just noticed that the tests failed in Travis but the fails seem rather weird to me... It passes on F30, works correctly locally.